nrf_security/doc/configuration.rst

```diff
@@ -7,14 +7,26 @@ Configuration
    :local:
    :depth: 2
 
-You can enable the Nordic Security Module using `PSA driver support`_ or with `Legacy crypto support`_.
+You can enable the Nordic Security Module using `PSA crypto support`_ or with `Legacy crypto support`_.
 
-PSA driver support
+PSA crypto support
 ******************
 
-To enable Nordic Security Module with the PSA driver support, set the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option along with additional configuration options, as described in :ref:`nrf_security_driver_config`.
+To enable Nordic Security Module with the PSA crypto support, set the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option along with additional configuration options, as described in :ref:`nrf_security_driver_config`.
```

Comment:
```suggestion
To enable the Nordic Security Module with the PSA crypto support, set the :kconfig:option:`CONFIG_NRF_SECURITY` Kconfig option along with additional configuration options, as described in :ref:`nrf_security_driver_config`.
```

Reply:
Do we have a common understanding of what 'setting a config option' means?

Are these options there to begin with, or does the user need to add them AND set them?

If the additional configuration is truly optional, I would suggest saying 'any additional configuration options'.

Reply:
Same note about definite article here with 'the PSA crypto support'. Support is an abstract concept in this sense, so I wouldn't expect it to have an article.